docs/configuration.html

```diff
@@ -255,6 +255,16 @@ <h3 class="anchor-heading"><a id="systemproperties" class="anchor-link"></a><a h
   Kafka supports some configuration that can be enabled through Java system properties. System properties are usually set by passing the -D flag to the Java virtual machine in which Kafka components are running.
   Below are the supported system properties.
   <ul class="config-list">
+    <li>
+      <h4><a id="org.apache.kafka.sasl.oauthbearer.allowed.urls"></a><a id="systemproperties_org.apache.kafka.sasl.oauthbearer.allowed.urls" href="#systemproperties_org.apache.kafka.sasl.oauthbearer.allowed.urls">org.apache.kafka.sasl.oauthbearer.allowed.urls</a></h4>
```

Comment:
Can we also update upgrade notes about the breaking change for existing clients.

Reply:
You're right! Updated.